backend/code/archipelag/event/models.py

```diff
@@ -2,31 +2,31 @@
 from django.db.models import DateTimeField
 from django.db.models import ForeignKey
 from django.db.models import Model
-from django.db.models import AutoField
-
+from django.db.models import PositiveIntegerField
 from django.contrib.postgres.fields import JSONField
 
 from archipelag.ngo.models import NgoUser
 
+SHARE = "SH"
+ADMIN_PROMOTE = "AP"
+REGISTER_FROM_INV = "RI"
+REGISTER = "R"
+EVENT_TYPES = (
+    (SHARE, 'SHARE'),
+    (ADMIN_PROMOTE, 'ADMIN PROMOTE'),
+    (REGISTER_FROM_INV, 'REGISTER FROM INVITATION'),
+    (REGISTER, 'REGISTER'),
+)
 
-class EventType(Model):
-    TYPES = (
-        ('SH', 'SHARE'),
-        ('AP', 'ADMIN PROMOTE'),
-        ('RI', 'REGISTER FROM INVITATION'),
-        ('R', 'REGISTER'),
-    )
-    type_id = AutoField(primary_key=True)
-    name = CharField(max_length=4, choices=TYPES)
-
-    def __str__(self):
-        return str(self.name)
 
 class Event(Model):
```

Comment:
I think, Event is too generic name. Maybe `EventLog`? or http://synonimy.woxikon.pl/en/event ?